components/atom/validationCode/src/index.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import {useState, useRef, createRef} from 'react'
+
+const DEFAULT_LENGTH = 6
+const BASE_CLASS = 'react-AtomValidationCode'
```

Comment:
All components have 'sui-ComponentName' as baseClass. Should we follow the convention?